app/browser/notifications/activityManager.js

```diff
@@ -107,6 +108,7 @@ function meetingStartNotifyHandler(self) {
 // eslint-disable-next-line no-unused-vars
 function myStatusChangedHandler(self) {
 	return async (event) => {
+		self.ipcRenderer.send('user-status-changed', { data: event.data })
 		if (PresenceState.isAway(event, self)) {
```

Comment:
@ewth this was an experiment. Now they're no more required. 
```js
if (PresenceState.isAway(event, self)) {
			await evaluateAndPreventAwayStatus(self, event);
		}
```
If you think that's going to be difficult for you to remove. I can do that and you can merge your changes.